src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -628,6 +628,24 @@ public bool PersistSecurityInfo
         }
         bool _persistSecurityInfo;
 
+        /// <summary>
+        /// Gets or sets a Boolean value that indicates if the Postgres DETAIL field is included in raised exceptions.
```

Comment:
as this is a suppression, not an inclusion, this description isn't quite accurate
Suggest swapping `included` with `suppressed`